src/Whoops/Run.php

```diff
@@ -395,6 +413,13 @@ private function writeToOutputNow($output)
         return $this;
     }
 
+    private static function isPHP7Throwable($exception)
+    {
+        return class_exists('Throwable', false) &&
```

Comment:
`Throwable` is a interface, therefore this should be 

``` php
interface_exists('Throwable', false)
```

instead, see https://3v4l.org/F2JGi

this might be the reason why it fails on travis right now.